scripts/check-workspace.sh

```diff
@@ -21,6 +21,9 @@ purely_std_crates=("lang/codegen" "metadata")
 also_wasm_crates=("env" "storage" "storage/derive" "allocator" "prelude" "primitives" "lang" "lang/macro" "lang/ir")
 all_crates=("${also_wasm_crates[@]}" "${purely_std_crates[@]}")
 
+# until https://github.com/rust-lang/cargo/issues/5034 has been implemented
```

Comment:
Same comment here

Reply:
I've added a link to our tracking issue and noted it down as a ToDo there.